pkg/pkg.ml

```diff
@@ -0,0 +1,7 @@
+#use "topfind"
+#require "topkg-jbuilder"
```

Comment:
should probably be `topkg-jbuilder.auto` as described in the docs. This will make topkg automatically lint all the top-level opam files and compile the project and run the tests on release (and other nice things).